agents/chwp/Dockerfile

```diff
@@ -15,5 +15,3 @@ ENTRYPOINT ["dumb-init", "python3", "-u", "hwpbbb_agent.py"]
 # Sensible default arguments
 CMD ["--site-hub=ws://crossbar:8001/ws", \
      "--site-http=http://crossbar:8001/call"]
-
-EXPOSE 8080/udp
```

Comment:
I thought this is something we should configure in the docker-compose.yml like following and we don't need to hard code in Dockerfile. 
https://github.com/simonsobs/socs/blob/develop/docs/agents/chwp_encoder.rst
```
ports:
    - "8081:8081/udp"
```
or 
```
expose:
  - "8081"
```
Agent is still working using these docker-compose.yml

Reply:
`EXPOSE` in the Dockerfile is more like documentation that the code in this container listens on port 8080, it doesn't actually publish the port (which is what the docker-compose syntax example you provided is doing.) I think it would be good to keep.